kessler/kessler.meta

```diff
@@ -78,85 +66,98 @@
   type = real | kind = kind_phys
   intent = in
 [ lyr_surf ]
-  standard_name = index_of_bottom_vertical_layer
+  standard_name = vertical_index_at_surface_adjacent_layer
   units = count
   dimensions = ()
   type = integer
   intent = in
 [ lyr_toa ]
-  standard_name = index_of_top_vertical_layer
+  standard_name = vertical_index_at_top_adjacent_layer
   units = count
   dimensions = ()
   type = integer
   intent = in
+[ cpair ]
+  standard_name = composition_dependent_specific_heat_of_dry_air_at_constant_pressure
+  units = J kg-1 K-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, vertical_layer_dimension)
+  intent = in
+[ rair ]
+  standard_name = composition_dependent_gas_constant_of_dry_air
+  units = J kg-1 K-1
+  type = real | kind = kind_phys
+  dimensions = (horizontal_dimension, vertical_layer_dimension)
+  intent = in
 [ rho ]
-  standard_name = dry_air_density
-  long_name = dry air density
+  standard_name = density_of_dry_air
+  long_name = density of dry air
   units = kg m-3
   dimensions = (horizontal_loop_extent, vertical_layer_dimension)
   type = real | kind = kind_phys
   intent = in
 [ z ]
-  standard_name = geopotential_height
-  long_name = geopotential height
+  standard_name = geopotential_height_wrt_surface
+  long_name = geopotential height w.r.t. local surface
   units = m
   dimensions = (horizontal_loop_extent, vertical_layer_dimension)
   type = real | kind = kind_phys
   intent = in
 [ pk ]
-  standard_name = inverse_exner_function_wrt_surface_pressure
-  long_name = inverse exner function w.r.t. surface pressure, (ps/p)^(R/cp)
-  units = count
+  standard_name = dimensionless_exner_function
+  long_name = exner function
+  units = 1
   dimensions = (horizontal_loop_extent, vertical_layer_dimension)
   type = real | kind = kind_phys
   intent = in
 [ theta ]
-  standard_name = potential_temperature
-  long_name = potential temperature
+  standard_name = air_potential_temperature
+  long_name = air potential temperature
   units = K
   dimensions = (horizontal_loop_extent, vertical_layer_dimension)
   type = real | kind = kind_phys
   intent = inout
 [ qv ]
   standard_name = water_vapor_mixing_ratio_wrt_dry_air
   long_name = mass mixing ratio of water vapor / dry air
-  advected = true
+  advected = True
```

Comment:
Just curious - does case matter for this variable?

Reply:
I don't think so, but in core python one must use `True` or `False`, so I decided it would be good to match that just in case.